api/src/main/java/net/kyori/adventure/title/Title.java

```diff
@@ -120,8 +120,24 @@
      * @param fadeOut the fade-out time
      * @return times
      * @since 4.0.0
+     * @deprecated for removal since 4.10.0, use {@link #times()}
      */
+    @ApiStatus.ScheduledForRemoval
```

Comment:
Perhaps add an `inVersion` attribute?

Reply:
Ooh yes I didn't even know this was a thing! Definitely add one of these with `5.0.0` as the version!